generative/maisi/README.md

```diff
@@ -0,0 +1,84 @@
+
+### 1. Installation
+Please refer to the [Installation of MONAI Generative Model](../README.md).
+
+Note: MAISI depends on [xFormers](https://github.com/facebookresearch/xformers) library, which unfortunately does not yet support ARM64.
+We will update after xFormers supports ARM64.
+
+### 2. Model inference and example outputs
+Please refer to [maisi_inference_tutorial.ipynb](maisi_inference_tutorial.ipynb) for the tutorial for MAISI model inference.
+
+### 3. Training example
+Training data preparation can be found in [./data/README.md](./data/README.md)
+
+#### [3.1 3D Autoencoder Training](./maisi_train_vae_tutorial.ipynb)
+
+Please refer to [maisi_train_vae_tutorial.ipynb](maisi_train_vae_tutorial.ipynb) for the tutorial for MAISI VAE model training.
+
+#### [3.2 3D Latent Diffusion Training](./scripts/diff_model_train.py)
+
+Please refer to [maisi_diff_unet_training_tutorial.ipynb](maisi_diff_unet_training_tutorial.ipynb) for the tutorial for MAISI VAE model training.
+
+#### [3.3 3D ControNet Training](./scripts/train_controlnet.py)
+
+We provide a [training config](./configs/config_maisi_controlnet_train.json) executing finetuning for pretrained ControlNet with a new class (i.e., Kidney Tumor).
+When finetuning with other new class names, please update the `weighted_loss_label` in training config
+and [label_dict.json](./configs/label_dict.json) accordingly. There are 8 dummy labels as placeholders in default `label_dict.json` that can be used for finetuning.
+Preprocessed dataset for ControNet training and more details anout data preparation can be found in the [README](./data/README.md).
+
+#### Training configuration
+The training was performed with the following:
+- GPU: at least 60GB GPU memory for 512 &times; 512 &times; 512 volume
+- Actual Model Input (the size of image embedding in latent space): 128 &times; 128 &times; 128
+- AMP: True
+
+#### Execute training:
+To train with a single GPU, please run:
+```bash
+python -m scripts.train_controlnet -c ./configs/config_maisi.json -t ./configs/config_maisi_controlnet_train.json -e ./configs/environment_maisi_controlnet_train.json -g 1
+```
+
+The training script also enables multi-GPU training. For instance, if you are using eight GPUs, you can run the training script with the following command:
+```bash
+export NUM_GPUS_PER_NODE=8
+torchrun \
+    --nproc_per_node=${NUM_GPUS_PER_NODE} \
+    --nnodes=1 \
+    --master_addr=localhost --master_port=1234 \
+    -m scripts.train_controlnet -c ./configs/config_maisi.json -t ./configs/config_maisi_controlnet_train.json -e ./configs/environment_maisi_controlnet_train.json -g ${NUM_GPUS_PER_NODE}
+```
+Please also check [maisi_train_controlnet_tutorial.ipynb](./maisi_train_controlnet_tutorial.ipynb) for more details about data preparation and training parameters.
+
+### 4. Questions and bugs
```

Comment:
[b->B]ugs

Reply:
I have updated the part. Please proceed. Thank you.